source/_integrations/whirlpool.markdown

```
@@ -51,9 +52,16 @@ Dryers:
 
 This {% term integration %} maps appliances to entities in Home Assistant. A single appliance may be represented by one or more entities.
 
+- [Binary Sensor](#binary_sensor)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)